--- manjaro-bamboo-splash/metadata.desktop	2022-12-13 15:14:11.363530010 +0700
+++ manjaro-bamboo-splash/metadata.new.desktop	2022-12-13 15:19:34.833523553 +0700
@@ -1,19 +1,19 @@
 [Desktop Entry]
-Comment=manjarobamboo
+Comment=cvm-ui-splash
 
 Encoding=UTF-8
 Keywords=Desktop;Workspace;Appearance;Look and Feel;Logout;Lock;Suspend;Shutdown;Hibernate;
-Name=Manjaro Bamboo
+Name=cvm-ui-splash
 
 Type=Service
 
 X-KDE-ServiceTypes=Plasma/LookAndFeel
 X-KDE-ParentApp=
-X-KDE-PluginInfo-Author=tazihad
+X-KDE-PluginInfo-Author=jiafeitech
 X-KDE-PluginInfo-Category=
 X-KDE-PluginInfo-Email=
-X-KDE-PluginInfo-License=GNU GPL v3
-X-KDE-PluginInfo-Name=manjarobamboo
+X-KDE-PluginInfo-License=MIT
+X-KDE-PluginInfo-Name=cvm-ui-splash
 X-KDE-PluginInfo-Version=
 X-KDE-PluginInfo-Website=
 X-KDE-fallbackPackage=org.kde.breeze.desktop
