--- src/manjaro-bamboo-splash/contents/splash/Splash.qml	2022-12-13 15:25:05.113517704 +0700
+++ src/manjaro-bamboo-splash/contents/splash/Splash.new.qml	2022-12-13 15:26:04.843516446 +0700
@@ -33,7 +33,7 @@
 
             anchors.centerIn: parent
 
-            source: "images/manjarologo.svg"
+            source: "images/logo.png"
 
             sourceSize.width: 254
             sourceSize.height: 50
